array/move_zeros_to_end.py

```diff
@@ -0,0 +1,24 @@
+"""
+Write an algorithm that takes an array and moves all of the zeros to the end, preserving the order of the other elements.
+move_zeros([false,1,0,1,2,0,1,3,"a"]) # returns[false,1,1,2,1,3,"a",0,0]
+
+The time complexity of the below algorithm is O(n).
+"""
+
+def move_zeros(array):
+    if len(array) < 1:
+        return array
+    else:
+        list = []
```

Comment:
avoid the use of built-in names (`list` in this case) as variables. Replace `list` without something like `result` or `res_array`.